@xen-orchestra/web/src/components/pool/connect/ConnectionForm.vue

```diff
@@ -38,16 +38,25 @@
       <RouterLink to="/">
         <UiButton accent="brand" size="medium" variant="secondary">{{ t('cancel') }}</UiButton>
       </RouterLink>
```

Comment:
I think we should fix this bad thing, when navigating with keyboard, we have two outlines, besides not being HTML compliant:

```suggestion
      <UiLink :to="{ name: '/(site)/dashboard'}" size="medium">
        {{ t('cancel') }}
      </UiLink>
```